### PR TITLE
FIXED parameter check and No log file and Display canvas

### LINKED ISSUE #121: Simulation gives SOMETHING WENT WRONG message before giving the results
When Simulate button is clicked, the message that appears is:

SOMETHING WENT WRONG. Please Check The Simulation Parameters And Diagram,

This should be replaced with:

Simulating... Please wait for output...

### PR DESCRIPTION
**Pull request by TheThinker277**
Fixes #121,#122,#126

### CODE REVIEW

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 1]
```diff
@@ -83,6 +85,8 @@ export default function SimulationScreen ({ open, close }) {
       if (loglines > 0) {
         console.log(loglines, 'log lines')
         loglines = 0
+      } else if (loglines === 0) {
+        setlog(true)
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]: remove these lines

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 2]
```diff
@@ -569,6 +573,7 @@ export default function SimulationScreen ({ open, close }) {
     sse.addEventListener('ERROR', e => {
       printloglines()
       console.log('ERROR', e)
+      seterror(1)
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]: change to setError('Error in simulation: ' + e.data)

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 3]
```diff
@@ -58,10 +58,12 @@ export default function SimulationScreen ({ open, close }) {
   const stitle = useSelector(state => state.netlistReducer.title)
   const taskId = useSelector(state => state.simulationReducer.taskId)
   const [isResult, setIsResult] = useState(false)
+  const [logs, setlog] = useState(false)
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]: remove this

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 4]
```diff
@@ -58,10 +58,12 @@ export default function SimulationScreen ({ open, close }) {
...
   const graphsRef = useRef([])
   const datapointsRef = useRef([])
   const timeoutRef = useRef(null)
   const [noOfGraphs, setNoOfGraphs] = useState(0)
+  const [error, seterror] = useState(0)
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]:
```suggestion
  const [error, setError] = useState('')
```
use camelCasing

set to empty string

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 5]
```diff
@@ -670,7 +675,8 @@ export default function SimulationScreen ({ open, close }) {
   }
 
   const typography1 = 'SOMETHING WENT WRONG. Please Check The Simulation Parameters.'
-  const typography2 = 'SOMETHING WENT WRONG. Please Check The Simulation Parameters And ' + process.env.REACT_APP_DIAGRAM_NAME + '.'
+  const typography2 = 'PLease Wait The Graph is Rendering with' + process.env.REACT_APP_DIAGRAM_NAME + '.'
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]:
```suggestion
  const typography2 = 'Please Wait. The Graph is Rendering with ' + process.env.REACT_APP_DIAGRAM_NAME + '...'
```
Small corrections:

- Please, not PLease
- full stop after Wait
- space after with

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 6]
```diff
@@ -670,7 +675,8 @@ export default function SimulationScreen ({ open, close }) {
...
+  const typography3 = 'NO LOG LINES FOUND'
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]: Remove this.

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 7]
```diff
@@ -743,9 +749,32 @@ export default function SimulationScreen ({ open, close }) {
...
+                 {
+                  (logs === true)
+                    ? <Grid item xs={12} sm={12}>
+                    <Paper className={classes.paper}>
+                      <Typography variant='h4' align='center' gutterBottom>
+                        {typography3}
+                      </Typography>
+                    </Paper>
+                  </Grid>
+                    : <span />
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]: Remove this

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 8]
```diff
@@ -743,9 +749,32 @@ export default function SimulationScreen ({ open, close }) {
                           : <div />
                       }
                       </>
-                    : (result.isGraph === 'true') ? <span>{typography1}</span> : <span />
+                    : (result.isGraph === 'false') ? <span>{typography1}</span> : <span />
+                }
+                {/* Diplay of Simulation parameter Not present */}
+                {
+                  (error === 1)
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]:
```suggestion
                  (error !== '')
```
change to check for nonempty string

File: blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js [hunk 9]
```diff
@@ -743,9 +749,32 @@ export default function SimulationScreen ({ open, close }) {
...
+                    ? <Grid item xs={12} sm={12}>
+                    <Paper className={classes.paper}>
+                      <Typography variant='h4' align='center' gutterBottom>
+                        {typography1}
```
Comment (sunilshetye) [inline blocks/eda-frontend/src/components/SchematicEditor/SimulationScreen.js]:
```suggestion
                        {error}
```
Change to the actual error message

**Review by sunilshetye — CHANGES_REQUESTED** [review]
remove log and typography3

change error to string. display it instead of typography3

**Comment by TheThinker277** [comment]
Fixes #121, #122

**Review by sunilshetye — APPROVED** [review]
good work

**Pull request merged**